servicepulse/intro-failed-messages.md

```diff
@@ -1,72 +1,93 @@
 ---
 title: Failed Message Monitoring
-summary: Describes how ServicePulse detects and monitors failed messages, and allows retrying, or archiving failing messages
+summary: Describes how ServicePulse detects and monitors failed messages, and allows retrying, or archiving failed messages
 component: ServicePulse
 tags:
 - ServicePulse
-reviewed: 2016-03-02
+reviewed: 2016-12-15
 related:
 - serviceinsight/managing-errors-and-retries
 ---
 
-When an NServiceBus endpoint fails to process a message, it performs a set of configurable attempts to recover from this failure. These attempts are referred to as "immediate retries" and "delayed retries" and in many cases allow the endpoint to overcome intermittent communication failures. For more details see [recoverability](/nservicebus/recoverability/).
+When an NServiceBus endpoint fails to process a message, it performs a set of configurable attempts to recover from this message failure. These attempts are referred to as "immediate retries" and "delayed retries" and in many cases allow the endpoint to overcome intermittent communication failures. See [recoverability](/nservicebus/recoverability/) for more details.
```

Comment:
I think the current text is better because it highlights the fact that auto retries are configurable.